src/modules/site-v2/templates/_includes/phenotype-database-table.html

```diff
@@ -204,9 +235,20 @@ <h5 id="offcanvas1Label"></h5>
     })
 
 
+    function search() {
+        const checkedTagsSelector = $(".filterCheckbox:checked")
+        let selectedTags = []
+        if (checkedTagsSelector.length) {
+            selectedTags = $(".filterCheckbox:checked").map(function() {
+                return $(this).val()
+            }).get()
+        }
+        const data = {csrf_token: $('#csrf_token').val(), search_val: $("#traitsSearch").val(), selected_tags: selectedTags}
+        sendPostRequest(data)
```

Comment:
we should look into moving from `callback` to `async` for these functions. In other words, make the `search` function async, and return the `await`-ed response; or throw an error. Then on the call site, handle the all the logistics.